packages/flutter/lib/src/material/radio.dart

```diff
@@ -23,9 +42,24 @@ class Radio<T> extends StatelessWidget {
     this.onChanged
   }) : super(key: key);
 
+  /// The value represented by this radio button.
   final T value;
+
+  /// The currently selected value for this group of radio buttons.
+  ///
+  /// This radio button is considered selected if its [value] matches the
+  /// [groupValue].
   final T groupValue;
+
+  /// The color to use when this radio button is selected.
+  ///
+  /// Defaults to accent color of the current theme.
   final Color activeColor;
+
+  /// Called when the user selects this radio button.
+  ///
+  /// For convenence, the radio button passes [value] as a parameter to this
+  /// callback.
```

Comment:
see comments about checkbox (that you need to update us accordingly, that if null we'll be disabled)